src/entity/MultipleChoiceQuestion.java

```diff
@@ -1,5 +1,26 @@
 package entity;
 class MultipleChoiceQuestion extends Question {
+    private Map<int, String> answerOptions = new HashMap<>();
+    private int correctAnswer;
 
+    public OpenEndedQuestion(){}
```

Comment:
This constructor should be MultipleChoiceQuestion right?

Reply:
oh yeah my bad will edit it right away thanks!!